examples/platform/silabs/efr32/rs911x/hal/efx32_ncp_host.c

```diff
@@ -179,8 +187,8 @@ sl_status_t sl_si91x_host_init(sl_si91x_host_init_configuration * config)
     GPIO_PinModeSet(WAKE_INDICATOR_PIN.port, WAKE_INDICATOR_PIN.pin, gpioModeWiredOrPullDown, 0);
 
     DMADRV_Init();
-    DMADRV_AllocateChannel(&rx_ldma_channel, NULL);
-    DMADRV_AllocateChannel(&tx_ldma_channel, NULL);
+    DMADRV_AllocateChannel((unsigned int *) &rx_ldma_channel, NULL);

```

Comment:
Better to use same data type **uint32_t** *?

Reply:
`DMADRV_AllocateChannel` function expects `unsigned int * rx_ldma_channel` , so without conversion it is giving an error .